koku/masu/test/database/test_aws_report_db_accessor.py

```diff
@@ -259,29 +261,39 @@ def test_bad_sql_execution(self):
             with self.assertRaises(ProgrammingError):
                 accessor._execute_processing_script("masu.database", script_file_path, {})
 
+    @retry(retries=settings.HIVE_PARTITION_DELETE_RETRIES)
+    def delete_ocp_on_aws_hive_partition_by_day(self, *args, **kwargs):
+        return self.accessor.delete_ocp_on_aws_hive_partition_by_day(*args, **kwargs)
+
```

Comment:
I'm not sure what the purpose of this method is. It's not a test and I don't see it being called the the test code.